CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -446,20 +525,25 @@ def validate_workbook(file, messages, args=None):
             for sheet in wbMissingSheets:
                 messages.append("%s is missing from the workbook." % (sheet,))
         for key in wsMismatchDict.keys():
-            messages.append("%s : %s row%s mismatched" % (key, wsMismatchDict[key], "" if wsMismatchDict[key]==1 else "s"))
-    return messages
+            messages.append("%s : %s row%s mismatched" %
+                            (key, wsMismatchDict[key], "" if wsMismatchDict[key] == 1 else "s"))
 
 
 def main(argv):
     args = parseArguments()
     messages = []
+    wb = None
     try:
-        validate_workbook(args.file, messages, args)
-    except xl.exceptions.InvalidFileException as e:
+        wb = xl.load_workbook(args.file)
+        if args.verbose:
+            print("Workbook Loaded")
```

Comment:
nit: using the `logging` library is generally more useful than having print as that can more easily be redirected to a file when using it as a library